pyteal/compiler/compiler_test.py

```diff
@@ -109,7 +109,7 @@ def test_compile_version_3():
 
 def test_compile_version_4():
```

Comment:
Could you add a `test_compile_version_5` check similar to this one?

Reply:
Sure, 1 sec